domains/manufacturing-environments/discover-behavior-specifications/q5.rq

```diff
@@ -0,0 +1,20 @@
+prefix : <https://purl.org/hmas/>
+prefix ex: <http://example.org/>
+prefix sh: <http://www.w3.org/ns/shacl#> 
+prefix prov: <http://www.w3.org/ns/prov#>
```

Comment:
Missing the prefix
`prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>`

Reply:
Updates on `interaction.ttl` addressed in https://github.com/HyperAgents/hmas/pull/136/commits/9c5862d03e5820ce3bb66d6c4548428fa2588b65.

Reply:
Updates on `q5.rq` addressed in https://github.com/HyperAgents/hmas/pull/136/commits/61b43f861adafcfaf37aeec7e7587a15084151b2.